README.md

```diff
@@ -90,6 +90,12 @@ One of our goals is to make the transition from ARM Templates to Bicep as easy a
 
 Note that while we want to make it easy to transition to Bicep, we will continue to support and enhance the underlying ARM Template JSON language. As mentioned in [What is Bicep?](#what-is-bicep), ARM Template JSON remains the wire format that will be sent to Azure to carry out a deployment.
 
+**What unique benefits do you get with Bicep?**
+a) Day 1 resource provider support. Any Azure resource whether in public preview or GA can be provisioned using Bicep b) No state or state files to manage. All state is stored in Azure, so makes it easy to collaborate and make changes to resources confidently. c) Tooling is the cornerstone to any great experience and our VSCode extension for Bicep makes it extremely easy to author and get started. d) Supported by Microsoft support and 100% free to use.
+
+**Why not use a programming language?**
+We spent a lot of time researching various different options and even prototyped a TypeScript based approach. We did over 120 customer calls, Microsoft Valued Professionals(MVP) conversations and collected quantitative data, and what we learnt was that in majority of organizations it was the cloud enablement teams that were responsible for provision the Azure infra and these folks were not familiar with programming languages and did not like that approach (as it had a steep learning curve). These users were our target users. In the future if we hear more feedback asking us to support a programming language approach, we are open to that as well. At the end of the day, we just want customers to be successful on Azure.
```

Comment:
`provision` should be `provisioning`

Reply:
We should also recommend Farmer or Pulumi here if that is what they are interested in.

Reply:
(or link to the section where we talk about them)

Reply:
Lastly, there is an additional layer of complexity when you model ARM resources in a programming language. You end up having to operate in two runtimes - the runtime of the programming language, where the things you can do are virtually limitless, and the ARM deployment runtime which is much more limited. The user-experience of trying to keep track of these two worlds proved to be very challenging. It's a subtle point, but very important to clarify.